lntest/itest/lnd_test.go

```diff
@@ -5060,6 +5063,104 @@ func testMultiHopSendToRoute(net *lntest.NetworkHarness, t *harnessTest) {
 	assertAmountPaid(t, "Alice(local) => Bob(remote)", net.Alice,
 		aliceFundPoint, amountPaid+(baseFee*numPayments), int64(0))
 
+	// Finally, we want to test allowing and disallowing of circular
+	// payments. Bob's node disallows circular payments and carol's node
+	// allows them. We will send payments over circular routes through
+	// each of these nodes.
```

Comment:
Wouldn't it be better to test this in a unit test like `TestSkipIneligibleLinksMultiHopForward`? To me it looks like there is nothing that an itest can exercise that can't be exercised in a lower level test.